mitmproxy/contentviews/__init__.py

```diff
@@ -170,7 +171,7 @@ def get_content_view(viewmode: View, data: bytes, **metadata):
 add(css.ViewCSS())
 add(urlencoded.ViewURLEncoded())
 add(multipart.ViewMultipart())
-add(image.ViewImage())
+add(pillow.ViewImage())
```

Comment:
We can rename `pillow.py` to `view.py` or something like this and then add this to `__init__.py`:

```
from .view import ViewImage
```

Then all the imports should work just as before and this file can stay untouched.